### ISSUE TITLE
[launching-pad] Find a home for the Embedded Working Group

### ISSUE THREAD

**Issue by jamesmunns** [rust-lang/leadership-council#127]
This is a tracking issue for finding a long-term home for the WG-embedded team (and all sub-teams).

Leads:

* @adamgreig
* @therealprof

Hey there all, as part of #118, I'm starting the triage process of trying to find permanent homes for teams currently under the launching pad. As WG-embedded is also a working group, #91 is also relevant.

In this first pass, I'm making tracking issues for each of the teams, starting the discussion on whether y'all think there is a reasonable team you think would make sense to move under, and in general checking whether the members of these teams are still actively working on things related to these teams.

I know that the embedded WG is still active, and there have been some prior discussions about how to potentially move some pieces into existing Rust teams, so there's no immediate action required (I'll circle back after the first pass on Launching Pad teams to see if any are inactive), but feel free to discuss or ask any questions here.

Thanks!

**Comment by jackh726** [comment]
I've mentioned it before at various times to different people, but I want to reiterate my thoughts here for posterity:

I think _as a whole_ the embedded working group is probably three different things (and please correct me if I am wrong, but maybe a longer form discussion is better had elsewhere):
1) A centralized place for people doing embedding things in Rust to meet and discuss.
2) A group to maintain various libraries/crates for doing embedded work (this may be either *architecture* based libraries like https://github.com/rust-embedded/riscv, or general libraries like https://github.com/rust-embedded/heapless)
3) A group to advocate to various Project teams for things to improve the embedded experience.

At the end of the day, I think these three parts of the wg probably can and should end up in different places:
1) I don't think having a centralized place to meet needs to be covered under the Project. There are other such groups (maybe gamedev) that probably *also* want similar types of space. We should consider what exactly is *needed* here and see if we can get some kind of dedicated funding/endorsement from the Foundation as-needed.
2) Libraries are interesting, because I think it's very reasonable to have these be "official" rust-lang crates (*especially* architecture-based libraries), but could also imagine they don't *need* to be. I'd be really happy to see a libs or libs/compiler subteam with the delegated responsibility to make these (as decided by the subteam) libs "official" and maintain them.
3) Advocacy is much harder and I have broader thoughts on how we should do this as a Project overall. Really though, I don't think the embedded wg *today* does much to enable this (not the fault of the wg, just how it is). The most simple thing we could start with is for teams to have an explicit way (and desire) for problems to be raised (which, most if not all do, but we can probably do better).

**Comment by jamesmunns** [comment]
Noting a couple of extra things that the WG does that might be relevant:

* maintaining "standard interface" crates, like:
    * [embedded-hal](https://docs.rs/embedded-hal/latest/embedded_hal/)
    * [embedded-hal-async](https://docs.rs/embedded-hal-async/latest/embedded_hal_async/)
    * [embedded-io](https://docs.rs/embedded-io/latest/embedded_io/)
* Acting as target maintainers for `thumb*` targets, and `riscv32i*` targets

**Comment by jackh726** [comment]
Yes, the first falls into (2) in my mind. The second I didn't quite cover, but I can imagine that is trivially covered by a marker team under compiler (similar to other marker teams like "apple" for people interested in some set of targets).
